src/org/opendatakit/aggregate/parser/BaseFormParserForJavaRosa.java

```diff
@@ -111,12 +116,29 @@ private static final void initializeJavaRosa() {
              // new CoreModelModule().registerModule();
              // replace with direct call to PrototypeManager
              PrototypeManager.registerPrototypes(SERIALIABLE_CLASSES);
+             redirectOutput();
```

Comment:
> Since this call changes System.out and .err application wide...
It doesn't. It redirects output from the JavaRosa library only. It only needs to be redirected before the library is used (and generates output).